library/aerospike

```diff
@@ -1,6 +1,6 @@
 # maintainer: Lucien Volmar <lucien@aerospike.com> (@volmarl)
 
 3.13.0.11: git://github.com/aerospike/aerospike-server.docker@29be954f3d764ae15f0666c39fc14b55f13f1033
-3.16.0.6: git://github.com/aerospike/aerospike-server.docker@961f91929117b71975382ae795c54ea19426bc47
-4.3.0.6: git://github.com/aerospike/aerospike-server.docker@e0a3ec717e07636b4aa032f31fe1aed62ef6039b
-latest: git://github.com/aerospike/aerospike-server.docker@e0a3ec717e07636b4aa032f31fe1aed62ef6039b
+4.1.0.6: git://github.com/aerospike/aerospike-server.docker@cbc02a818af8ea2afb893093d062339a8b851e6d
```

Comment:
This commit does not seem to exist, did you forget to push it?

Reply:
These two seems to exist:

https://github.com/aerospike/aerospike-server.docker/commit/cbc02a818af8ea2afb893093d062339a8b851e6d

https://github.com/aerospike/aerospike-server.docker/commit/d2fa950e7cb5c10ccc39d778b2aef233440a855a